_source/_posts/2022-05-20-java-rest-api-quarkus.adoc

```diff
@@ -0,0 +1,324 @@
+---
+layout: blog_post
+title: "Kickstart a Java REST API with Quarkus"
```

Comment:
@havi I originally had "Build a Secure Java REST API with Quarkus" as the title.

I changed it because if you look at the main blog page, it will come only a post or two after "Build Secure Ionic Apps with Angular and JHipster". It seems the titles might be a bit too close. I also figured maybe we could use a different word for each framework. For example:

- Make a Java REST API with Micronaut
- Start a Java REST API with Spring Boot
- Hack Together a Java REST API with Helidon 😅

OK, "Build" might be better...